libretroshare/src/pqi/authssl.cc

```diff
@@ -1233,8 +1243,38 @@ int AuthSSLimpl::VerifyX509Callback(int /*preverify_ok*/, X509_STORE_CTX* ctx)
 		return verificationFailed;
 	}
 
+	bool isSslOnlyFriend = false;
+
+    // For SSL only friends (ones added through short invites) we check that the fingerprint
+    // in the key (det.gpg_id) matches the one of the handshake.
+	{
+		RsPeerDetails det;
+
+		if(rsPeers->getPeerDetails(sslId,det))
+			bool isSslOnlyFriend = det.skip_pgp_signature_validation;
```

Comment:
`bool` must be removed otherwise we are redefining and thus shadowing `isSslOnlyFriend` in that scope and not assigning the variable defined at line 1246

Reply:
thx.